spec/app.spec.js

```diff
@@ -1,12 +1,22 @@
 // @flow
 import request from 'supertest';
 import app from '../src/app';
+import GetArtifactRequest from './Get_Artifacts_Request';
```

Comment:
Files without snake case imo. Also maybe if you'll have more sample requests later, you could group them all in one file? Don't really see the point of splitting them. Nobody will check these requests anyways I think. So rename the file to something like 'SampleRequests'?